### PR TITLE
Add integration tests for `AppConfigRepository`

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.shortcut.com/simpledotorg/story/4885/add-instrumentation-tests-for-manifest-and-states-api

### CODE REVIEW

**Comment by msasikanth** [comment]
Since the Android tests on GitHub are broken, do run them locally for now.

File: app/src/androidTest/java/org/simple/clinic/appconfig/AppConfigRepositoryAndroidTest.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    // when
+    val manifestResult = appConfigRepository
+        .fetchAppManifest()
+        .blockingGet()
+
+    // then
+    assertThat(manifestResult is FetchSucceeded).isTrue()
+  }
+
+  @Test
```
Comment (janhavisinghh) [inline app/src/androidTest/java/org/simple/clinic/appconfig/AppConfigRepositoryAndroidTest.kt:75]: two integration tests seem to be failing. Can you ptal?

**Review by janhavisinghh — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/org/simple/clinic/appconfig/AppConfigRepositoryAndroidTest.kt (hunk 1 above)
Comment (janhavisinghh) [reply app/src/androidTest/java/org/simple/clinic/appconfig/AppConfigRepositoryAndroidTest.kt:75]: `fetching_states_for_selected_country_should_work_correctly` and `fetching_countries_from_manifest_should_work_correctly`

Comment (msasikanth) [reply app/src/androidTest/java/org/simple/clinic/appconfig/AppConfigRepositoryAndroidTest.kt:75]: Did they fail locally? It's working fine for me locally and I don't see any failures (for this) on CI 🤔

Comment (janhavisinghh) [reply app/src/androidTest/java/org/simple/clinic/appconfig/AppConfigRepositoryAndroidTest.kt:75]: umm yeah I tested locally 😅

Comment (janhavisinghh) [reply app/src/androidTest/java/org/simple/clinic/appconfig/AppConfigRepositoryAndroidTest.kt:75]: oh it's fine. There was some issue from my side. nvm

**Review by janhavisinghh — APPROVED** [review]

**Pull request merged**